rest_framework-stubs/test.pyi

```diff
@@ -81,14 +81,18 @@ class APIClient(APIRequestFactory, DjangoClient):
 
 class APITransactionTestCase(testcases.TransactionTestCase):
     client_class: Type[APIClient] = ...
+    client: APIClient
 
 class APITestCase(testcases.TestCase):
     client_class: Type[APIClient] = ...
+    client: APIClient
 
 class APISimpleTestCase(testcases.SimpleTestCase):
     client_class: Type[APIClient] = ...
+    client: APIClient
 
 class APILiveServerTestCase(testcases.LiveServerTestCase):
     client_class: Type[APIClient] = ...
+    client: APIClient
 
 class URLPatternsTestCase(testcases.SimpleTestCase): ...
```

Comment:
What about this case?

Reply:
`URLPatternsTestCase` does not define a custom `client` as far as I can tell:
https://github.com/encode/django-rest-framework/blob/e5fb9af0eaffde683fa0af3987085f86cf0d2640/rest_framework/test.py#L368